<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QObject#centralwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0795455 rgba(0, 57, 70, 255), stop:0.5625 rgba(0, 85, 104, 255), stop:0.954545 rgba(0, 107, 131, 255));}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1101</width>
        <height>591</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1101</width>
        <height>591</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QObject#frame{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.0113636 rgba(113, 181, 190, 255));
	
border-radius:20px;}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>9</y>
         <width>681</width>
         <height>45</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>681</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>681</width>
         <height>45</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="refreshLayersEnrich">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Refresh Layers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="chooseLayerEnrich">
            <property name="minimumSize">
             <size>
              <width>321</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>321</width>
              <height>21</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;

border-color:rgb(45, 45, 45);

background-color: rgb(253, 252, 250);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadLayerEnrich">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Load Layer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addEnrichedLayerButton">
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>121</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Add enriched layer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>111</y>
         <width>1065</width>
         <height>471</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1059</width>
         <height>471</height>
        </size>
       </property>
       <widget class="QTableWidget" name="enrichTable">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>1041</width>
          <height>451</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1041</width>
          <height>451</height>
         </size>
        </property>
       </widget>
       <widget class="QTableWidget" name="enrichTableResult">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1041</width>
          <height>451</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1041</width>
          <height>451</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>60</y>
         <width>481</width>
         <height>45</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>481</width>
         <height>45</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addEnrichedLayerRowButton">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Add Row</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="whattoenrich">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>What to enrich?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="startEnrichment">
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>121</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 127, 165);
border-radius:10px;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Start Enrichment</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
